extensions/spring-data-jpa/deployment/src/main/java/io/quarkus/spring/data/deployment/SpringDataJPAProcessor.java

```diff
@@ -53,6 +55,27 @@ void build(CombinedIndexBuildItem index,
             BuildProducer<GeneratedBeanBuildItem> generatedBeans,
             BuildProducer<AdditionalBeanBuildItem> additionalBeans) {
 
+        Config config = ConfigProvider.getConfig();
```

Comment:
Maybe let's make it an entirely separate build step?

Reply:
Do you mean a `@BuildStep` method? Why? I would make it in a separate `private` method but this actions don't `consume` either `produce` any items for others `steps` so I don't see why make it in a `@BuildStep`, makes sense?

Reply:
@gsmet any follow-up on @aureamunoz's comment?

Reply:
I've just seen your comment @geoand :woman_facepalming:
@gsmet did you see my comment? WDYT?